### PR TITLE
231 add buy screen

### PR DESCRIPTION
**Pull request by IvanPletinskiy**
image

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -52,11 +53,13 @@ function App() {
             .then((withdrawals) => setWithdrawals(withdrawals))
     }, []);
 
+    let navigate = useNavigate();
```
Comment (IvanPletinskiy) [inline frontend/src/App.js:56]: Must be called within function component and not class

File: frontend/src/components/AccountsList.js [hunk 2]
```diff
@@ -13,7 +11,19 @@ class AccountsList extends React.Component {
                     </div>
                 } else {
                     buttons = <div>
-                        <button>Купить</button>
+                        <button
+                            onClick={() => {
+                                navigate("/buy", {
```
Comment (IvanPletinskiy) [inline frontend/src/components/AccountsList.js:16]: Navigate and pass args

File: frontend/src/routes/buy/BuyScreen.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+function BuyScreen() {
+    let navigate = useNavigate();
+
+    const [amount, setAmount] = useState(' ');
+
+    const [options, setOptions] = React.useState([{value: 'chocolate', label: 'Chocolate'}, {
+        value: 'strawberry', label: 'Strawberry'
+    }, {value: 'vanilla', label: 'Vanilla'},]);
+
+    const {state} = useLocation();
```
Comment (IvanPletinskiy) [inline frontend/src/routes/buy/BuyScreen.js:15]: Retrieve passed args

File: frontend/src/routes/buy/BuyScreen.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+            })
+    }, []);
+
+    const [selectedOption, setSelectedOption] = useState(null);
+    const targetAccountText = ca_number + " " + getCurrencyNameById(fk_cc_id)
+    const [youWillBuyText, setYouWillBuyText] = useState("");
+
+    return (<div>
+        <button onClick={() => {
+            navigate(-1);
```
Comment (IvanPletinskiy) [inline frontend/src/routes/buy/BuyScreen.js:37]: Navigate back

**Review by Weizhau — APPROVED** [review]

**Pull request merged**